app/views/registrations/register.html.erb

```diff
@@ -77,7 +77,7 @@
             </h4>
           </div>
           <div class="panel-body">
-            <% if @registration.stripe_payment_intents.processing.any? %>
+            <% if @registration.payment_intents.unpaid.any? %>
```

Comment:
See note about `unpaid` and whether it is an appropriate replacement for `processing`

Reply:
(same logic as in my other comment applies)

Reply:
In which case, we're literally ONLY looking for the Stripe `processing` status? Presumably because for `processing` we want to show them that they have a payment busy processing, but in all other cases we just want to render the payment form? 

I'm a little confused why `processing` specifically is used here, when the general logic seems quite similar to the above case (where `pending` was used before)